src/main/java/org/fenixedu/treasury/util/TreasuryBootstrapper.java

```diff
@@ -114,7 +114,6 @@ public static void bootstrap(String institutionName, String institutionInitials,
 
         // On create of finantialInstitution will be called TreasuryBootstrap.bootstrapFinantialInstitution(FinantialInstitution finantialInstitution)
         initializeFinantialInstituition(institutionName, institutionInitials, countryCode);
-
```

Comment:
@joaofreitas-qubit we can remove this change, since it's just a formatting issue